### DISCUSSION TITLE
No standard commands found after installing home-brew

### DISCUSSION THREAD

**Discussion by JJOC0205** [Homebrew/discussions#3658] [Getting started] [answered]
Hi there, new to home-brew and MacOS as a whole. So I recently installed home-brew onto my computer and since installing it I get this message when I open up my terminal

/etc/zshrc:7: command not found: locale
/etc/zshrc_Apple_Terminal:104: command not found: mkdir

When I try any command that's not a brew one(ls,cd,mkdir,sudo), it gives me this: 'zsh: command not found: ls'
Can anyone help me figure this out? Its been hard to get my schoolwork done since I did this as I use the terminal quite often and I've resorted to using my old windows machine since this happened

**Comment by carlocab** [comment, marked as answer]
What's the output of
```
/usr/bin/grep brew ~/.zprofile
/bin/cat ~/.zprofile
```
?

**Reply by JJOC0205** [reply]
`export PATH="$PATH:/Users/jonathonoconnell/Library/Application Support/JetBrains/Toolbox/scripts"eval $(/opt/homebrew/bin/brew shellenv)
eval $(/opt/homebrew/bin/brew shellenv)
eval "$(/opt/homebrew/bin/brew shellenv)"`

`# Added by Toolbox App

export PATH="$PATH:/Users/jonathonoconnell/Library/Application Support/JetBrains/Toolbox/scripts"eval $(/opt/homebrew/bin/brew shellenv)
eval $(/opt/homebrew/bin/brew shellenv)
eval "$(/opt/homebrew/bin/brew shellenv)"`

I'd never gotten any message about the JetBrains Toolbox before, could that have caused the issue?

Ive never used the discussion part of Github before so im not sure how to get the nice box like you have

**Reply by carlocab** [reply]
Do
```
/usr/bin/open ~/.zprofile
```
This will open the file in TextEdit. You should clean this up to look like
```
# Added by Toolbox App
export PATH="$PATH:/Users/jonathonoconnell/Library/Application Support/JetBrains/Toolbox/scripts"
eval "$(/opt/homebrew/bin/brew shellenv)"
```
and remember to save your changes. Restart your Terminal for the changes to take effect.

**Reply by JJOC0205** [reply]
When I try to it says:
`zsh: command not found: open
`

**Reply by carlocab** [reply]
The issue is that when Toolbox App edited your `~/.zprofile`, it didn't leave a new line at the end of the file, so appending to `~/.zprofile` doesn't work. It really should have left a new line there. (Our install instructions should probably also work even in the above case, but this is typically quite rare.)

**Reply by carlocab** [reply]
> When I try to it says:
> `zsh: command not found: open`

Sorry, I forgot your `PATH` was messed up. Do
```
/usr/bin/open ~/.zprofile
```

**Reply by JJOC0205** [reply]
Thank you so much for your help, you have no idea how grateful I am. You are a true lifesaver. Ive been searching for a answer online for a bit and none of them were as helpful as you. Hope you have a good day!!

**Reply by MohamedAboElM3aTy** [reply]
/Users/m/.zshrc:104: command not found: ‘eval
‘eval ”’
/Users/m./.zshrc:106: command not found: ‘eval
‘eval ”’
/Users/m./.zshrc:108: command not found: ‘eval
/Users/m./.zshrc:109: command not found: echo‘eval


I trying to export this =>  echo‘eval "$(pyenv init —path)”’>> ~/.zprofile\necho‘eval "$(pyenv init-)”’ >> ~/.zshrc 
but from that moment when i ope =n the terminal it shows me that error
